hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -137,6 +137,26 @@ if [ `cat /etc/hosts | grep ${api_ip} | grep "default-route-openshift-image-regi
   sed -i "s/${api_ip}/${api_ip} default-route-openshift-image-registry.apps.${cluster_name}.${domain_name}/g" /etc/hosts
 fi
 
+# Create performance profile to allocate hugepages
+cat <<EOF | oc apply -f -
+apiVersion: performance.openshift.io/v1
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  cpu:
```

Comment:
this has no equivalent in `run-e2e-conformance-virtual-cluster`  i guess this translates to kernel `isolcpus` and kubelet cpu config.

no objection in keeping, but does it have any benefit in the e2e test env ?